es-vue-base/src/lib-components/EsReviewModal.vue

```diff
@@ -14,7 +14,7 @@
             class="modal-container">
             <b-row>
                 <b-col
-                    class="p-0"
+                    class="p-0 pr-3"
```

Comment:
Two things here:
1. The single-number naming scheme for spacing classes is [deprecated](https://design.energysage.dev/atoms/spacing); you'll want to use `pr-100` here instead.
2. You probably don't need the extra padding on mobile; only at the `lg` breakpoint when this section, already starting to appear side-by-side at the `md` breakpoint, is reordered to the left side without any spacing between it and the column to its right. So you can make this `pr-lg-100` so it only starts happening at the `lg` breakpoint and up.

Also, the amount of spacing is up to you, your team, and your designer, but I might recommend doubling it to `pr-lg-200` just to give everything a bit more breathing room.

Reply:
@ericdouglaspratt thank you so much for your feedback. I have user `pr-lg-200` as recommended by you. Our designer didn't ask for a particular number, just enough padding to separate the two sub-components so `pr-lg-200` works perfectly!